modules/rostests/apitests/gdi32/GetTextMetrics.c

```diff
@@ -0,0 +1,518 @@
+/*
```

Comment:
[We changed our header style to SPDX.](https://git.reactos.org/?p=reactos.git;a=blob;f=dll/appcompat/apphelp/apphelp.c)

Reply:
OK.